strata_period_method.py

```diff
@@ -35,7 +35,9 @@ def lambda_handler(event, context):
 
         logger.info("Vaildated params")
 
-        input_data = pd.DataFrame(event)
+        input_data = pd.DataFrame(event["data"])
+        survey_column = event["survey_column"]
```

Comment:
Is this to be changed to the correct name or is it actually called survey_column in the data? As in enrichent it's simply called 'survey'.